broker-daemon/broker-rpc/wallet-service/commit.js

```diff
@@ -66,15 +60,26 @@ async function commit ({ params, relayer, logger, engines, orderbooks }, { Empty
   if (MINIMUM_FUNDING_AMOUNT.gt(balanceInCommonUnits)) {
     throw new PublicError(`Minimum balance of ${MINIMUM_FUNDING_AMOUNT} needed to commit to the relayer`)
   } else if (maxChannelBalance.lt(balance)) {
+    const maxChannelBalanceInCommonUnits = Big(maxChannelBalance).div(engine.quantumsPerCommon).toString()
     logger.error(`Balance from the client exceeds maximum balance allowed (${maxChannelBalance.toString()}).`, { balance })
-    throw new PublicError(`Maximum balance of ${maxChannelBalance.toString()} exceeded for committing of ${balance} to the relayer. Please try again.`)
+    throw new PublicError(`Maximum balance of ${maxChannelBalanceInCommonUnits} ${symbol} exceeded for committing of ${balanceInCommonUnits} ${symbol} to the Relayer. Please try again.`)
+  }
+
+  // Also check that the inbound channel does not exceed the channel maximum, otherwise our channel will succeed, but our request to the Relayer will fail.
```

Comment:
[nit] comment line is too long (we should shorten the other lines in this file, but its OK for now)